Relationships/RelationshipResolver.cs

```diff
@@ -1,284 +1,422 @@
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using log4net;
 
 namespace CKAN
 {
-    public class RelationshipResolverOptions : ICloneable
-    {
-        public bool with_all_suggests;
-        public bool with_recommends = true;
-        public bool with_suggests;
-        public bool without_toomanyprovides_kraken = false;
-        public bool without_enforce_consistency = false;
-        public object Clone()
-        {
-            return MemberwiseClone();
-        }
-    }
-
-    // Alas, it appears that structs cannot have defaults. Try
-    // DefaultOpts() to get friendly defaults.
-
-    // TODO: RR currently conducts a depth-first resolution of requirements. While we do the
-    // right thing in processing all depdenencies first, then recommends, and then suggests,
-    // we could find that a recommendation many layers deep prevents a recommendation in the
-    // original mod's recommends list.
-    //
-    // If we resolved in things breadth-first order, we're less likely to encounter surprises
-    // where a nth-deep recommend blocks a top-level recommend.
-
-    public class RelationshipResolver
-    {
-        // A list of all the mods we're going to install.
-        private static readonly ILog log = LogManager.GetLogger(typeof(RelationshipResolver));
-        private readonly Dictionary<string, CkanModule> modlist = new Dictionary<string, CkanModule>();
-        private Registry registry;
-        private KSPVersion kspversion;
-
-        
+	public class RelationshipResolverOptions : ICloneable
```

Comment:
Is it just me, or have our indent levels changed somewhat? I fear that some of the styling configuration may have been lost during the Great Repo Split of 2014.